### DISCUSSION TITLE
Killer instincts N64plus fz

### DISCUSSION THREAD

**Discussion by Bioshock305** [mupen64plus-ae/mupen64plus-ae#918] [General]
When playing arcade.... when I get to the player glacius... the game freezes n I'm force to restart the game... bug?

**Comment by fzurita** [comment]
Strange, I'm not seeing the issue here with the playstore or development version. It could be related to your phone. Which phone do you have? 

Can you try a different emulation profile? To do that, select the game, go to settings, and change the emulation profile to something like GLideN64-Medium.

**Comment by Bioshock305** [comment]
Have a s20 samsung n it never happend till after the recent update....



-------- Original message --------
From: Francisco Zurita ***@***.***>
Date: 7/21/21 10:47 PM (GMT-06:00)
To: mupen64plus-ae/mupen64plus-ae ***@***.***>
Cc: Bioshock305 ***@***.***>, Author ***@***.***>
Subject: Re: [mupen64plus-ae/mupen64plus-ae] Killer instincts N64plus fz (#918)


Strange, I'm not seeing the issue here with the playstore or development version. It could be related to your phone. Which phone do you have?

Can you try a different emulation profile? To do that, select the game, go to settings, and change the emulation profile to something like GLideN64-Medium.

**Comment by fzurita** [comment]
Ok, are you in the absolute latest version? I've been updating like once a week.

**Comment by Bioshock305** [comment]
Yes sir.... n I've been checking daily.... just to make sure... cause I wasn't sure when they do updates till now... that u said  once a week... but im am glad u wrote back really



-------- Original message --------
From: Francisco Zurita ***@***.***>
Date: 7/21/21 10:52 PM (GMT-06:00)
To: mupen64plus-ae/mupen64plus-ae ***@***.***>
Cc: Bioshock305 ***@***.***>, Author ***@***.***>
Subject: Re: [mupen64plus-ae/mupen64plus-ae] Killer instincts N64plus fz (#918)


Ok, are you in the absolute latest version? I've been updating like once a week.

**Comment by fzurita** [comment]
Updates are a manual process and I have been trying to stick to one a week. So it's by no means a guarantee. 

So, I'm unable to reproduce the issue. I have a similar phone too, galaxy s21.

Which exact version of the game are you using? The game has server versions. Which emulation profile are you using?

**Comment by Bioshock305** [comment]
From glide accurate... to glide64 medium....m they all freeze when u get to glacius..... some might allow a few more secs.... befor freezing





-------- Original message --------
From: Francisco Zurita ***@***.***>
Date: 7/21/21 11:10 PM (GMT-06:00)
To: mupen64plus-ae/mupen64plus-ae ***@***.***>
Cc: Bioshock305 ***@***.***>, Author ***@***.***>
Subject: Re: [mupen64plus-ae/mupen64plus-ae] Killer instincts N64plus fz (#918)


Updates are a manual process and I have been trying to stick to one a week. So it's by no means a guarantee.

So, I'm unable to reproduce the issue. I have a similar phone too, galaxy s21.

Which exact version of the game are you using? The game has server versions. Which emulation profile are you using?

**Comment by Bioshock305** [comment]
App version 3.0.288beta free
Code.288


-------- Original message --------
From: Francisco Zurita ***@***.***>
Date: 7/21/21 11:10 PM (GMT-06:00)
To: mupen64plus-ae/mupen64plus-ae ***@***.***>
Cc: Bioshock305 ***@***.***>, Author ***@***.***>
Subject: Re: [mupen64plus-ae/mupen64plus-ae] Killer instincts N64plus fz (#918)


Updates are a manual process and I have been trying to stick to one a week. So it's by no means a guarantee.

So, I'm unable to reproduce the issue. I have a similar phone too, galaxy s21.

Which exact version of the game are you using? The game has server versions. Which emulation profile are you using?

**Comment by fzurita** [comment]
Ok, which version of the ROM?

**Comment by fzurita** [comment]
By rom version I mean EU, USA 1.0, 1.1, or 1.2. I can't really be fixed if it can't be reproduced.

**Comment by Bioshock305** [comment]
[cid:storage_emulated_0__EmailTempImage_HEV_1627005696654_jpg_1627005696746]



-------- Original message --------
From: Francisco Zurita ***@***.***>
Date: 7/22/21 7:26 PM (GMT-06:00)
To: mupen64plus-ae/mupen64plus-ae ***@***.***>
Cc: Bioshock305 ***@***.***>, Author ***@***.***>
Subject: Re: [mupen64plus-ae/mupen64plus-ae] Killer instincts N64plus fz (#918)


By rom version I mean EU, USA 1.0, 1.1, or 1.2. I can't really be fixed if it can't be reproduced.
